backend/extensions/Squidex.Extensions/Actions/SignalR/SignalRActionHandler.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Azure.SignalR.Management;
+using Squidex.Domain.Apps.Core.HandleRules;
+using Squidex.Domain.Apps.Core.Rules.EnrichedEvents;
+
+namespace Squidex.Extensions.Actions.SignalR
+{
+    public sealed class SignalRActionHandler : RuleActionHandler<SignalRAction, SignalRJob>
+    {
+        private readonly ClientPool<(string ConnectionString, string HubName), IServiceManager> clients;
+        public SignalRActionHandler(RuleEventFormatter formatter)
```

Comment:
Please add newline after fields.

Reply:
Indeed, it was just a test, I'm switching back to CamelCase